.github/workflows/e2e-tests.yml

```diff
@@ -0,0 +1,76 @@
+name: Deploy
+
+on:
+  pull_request:
+    paths:
+      - "packages/apps/**"
```

Comment:
Por que só vai rodar em pull request e por que só nos apps ?
Também depende do `packages/modules` e esse por sua vez depende de outras paradas ainda...
Talvez seja mais fácil ouvir todos os pacotes adicionando algumas exceções (storefront, passport, mais?)

Reply:
E acho que tem que rodar no push em main também, com possibilidade de ser skipado com `[skip ci]`: https://github.com/ecomplus/storefront-starter/blob/master/.github/workflows/build-and-deploy.yml#LL23C40-L23C40